WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
Absolutely not. The AI sending foreign missionaries is directly tied to his AI_ZEAL rating. Religions not always known for prolestyzing should not _always_ be banned from dispathing missionaries. Rather, it should depend on if the ruler of the religion is zealous enough.

For example, the Khazars were unusual in their attempts to spread Judaism. However, Jews, historically, never prolestyzed their religion. Rather, they waited for converts to come to Judaism, and they in fact warned them that it is an awesome responsibility to follow the holy Law of God. I have spoken to an actual Jew about this before.